src/core.js

```diff
@@ -3,10 +3,11 @@ const percentageProgress = document.getElementById('percentage-progress')
 
 const today = new Date()
 const year = today.getFullYear()
+const isLeapYear = (year) => ((year % 4 === 0) && (year % 100 !== 0)) || (year % 400 === 0)
 
 const daysPerMonth = {
   1: 31,
-  2: year % 4 === 0 ? 29 : 28,
+  2: isLeapYear(year) ? 29 : 28,
```

Comment:
What about creating a `getFebruaryDays` function to extract this logic?

```suggestion
  2: getFebruaryDays(year),
```

with a function after the `isLeapYear`:

```javascript
const getFebruaryDays = (year) => isLeapYear(year) ? 29 : 28;
```

Reply:
I like it!